.pre-commit-config.yaml

```diff
@@ -1,5 +1,11 @@
 repos:
 - repo: https://github.com/psf/black
-  rev: 19.3b0
+  rev: 19.10b0
```

Comment:
This change causes the file to go out of sync with the [upstream source](/jaraco/skeleton). I wonder if it's possible to just somehow specify 'latest' or similar and avoid constantly bumping this value across hundreds of projects.

Reply:
No, you need to pin them and can run `pre-commit autoupdate` to update them.

I've not used it, but https://github.com/asottile/all-repos could help update many projects.